### ISSUE TITLE
ARIA listbox missing child roles

### ISSUE THREAD

**Issue by iantrudell** [ng-select/ng-select#951]
**Describe the bug**
Google Lighthouse Audits are at it again with Accessibility errors:

```
Elements with [role] that require specific children [role]s, are missing.

Some ARIA parent roles must contain specific child roles to perform their intended accessibility functions.
```

**To Reproduce** 
1. Add an ng-select with role="listbox"
2. Run a Google Chrome Lighthouse Audit
3. ng-select will fail Accessibility due to missing child roles from the listbox

**Expected behavior**
Passed Accessibility tests

**Screenshots**
![accessibility-audit](https://user-images.githubusercontent.com/33202657/48566055-e309b400-e8c7-11e8-8fcc-2b61bd832a81.PNG)

**Desktop (please complete the following information):**
 - OS: Windows, CentOS 7
 - Browser: Chrome
 - Version: 71

**Additional context**
https://www.w3.org/TR/wai-aria-practices-1.1/#Listbox
https://dequeuniversity.com/rules/axe/2.2/aria-required-children?application=lighthouse

**Comment by stijnwesterhof** [comment]
Having the same issue.. Got any fix?

**Comment by varnastadeus** [comment]
All roles are there, issue is that dropdown doesn't exist in the DOM until control is not opened. I suggest to ignore this warning.

**Comment by vivekshah-zymr** [comment]
@iantrudell @stijnwesterhof @varnastadeus 
Any update on this? I am facing the same issue.

**Comment by mcha-dev** [comment]
Error persists even with `[isOpen]=true`
